styles/globalStyle.ts

```diff
@@ -9,7 +9,7 @@ const COLORS = {
   deepSea: '#064aff',
   deepSea80: 'rgba(6, 135, 255, 0.8)',
   graphiteGrey: '#333333',
-  crystalWhite: '#ffffff',
+  crystalWhite: 'rgba(255, 255, 255)',
```

Comment:
colors should be in one style rgba(... of #fff...

Reply:
Converted